test/unit/imitates_type/makes_double/declares_dry_class_test.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
+
+module Mocktail
+  class DeclaresDryClassTest < Minitest::Test
+    include Mocktail::DSL
+
+    def setup
+      @subject = DeclaresDryClass.new
+    end
+
+    class Fib
+      def lie(truth, lies:)
+      end
+    end
+
+    def test_calls_handle_dry_call_with_what_we_want
+      fake_fib_class = @subject.declare(Fib, [:lie])
```

Comment:
Oh good you'll get something out of my talk on not using subject, even if you're not using RSpec.

Reply:
Sorry man I'm a hardcore fan of consistently calling the SUT subject